haskell/nettest/TransferlistIntegration.hs

```diff
@@ -0,0 +1,147 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+module TransferlistIntegration
+  ( scTransferlistIntegrationScenario
```

Comment:
If you manage to generalize `scTransferScenario`, this whole module should become unnecessary.